pkg/features/kube_features.go

```diff
@@ -434,6 +434,12 @@ const (
 	//
 	// Enables the OpenStack Cinder in-tree driver to OpenStack Cinder CSI Driver migration feature.
 	CSIMigrationOpenStack utilfeature.Feature = "CSIMigrationOpenStack"
+
+	// owner: @Huang-Wei
+	// alpha: v1.15
+	//
+	// Schedule pods evenly across available topology domains.
+	EvenPodsSpread utilfeature.Feature = "EvenPodsSpread"
```

Comment:
Is `EvenPodSpreading` a better name for the feature?

Reply:
I recalled @smarterclayton [mentioned](https://github.com/kubernetes/enhancements/pull/851#discussion_r274250936) that `ing` is not a good name. But I'm open to any reasonable name of the feature gate.

Reply:
Any suggestion on the naming?

cc/ @bsalamat @lavalamp @liggitt @smarterclayton